src/psyclone/tests/kernel_transformation_test.py

```diff
@@ -42,7 +42,8 @@
 from fparser.two.utils import walk_ast
 from dynamo0p3_build import Dynamo0p3Build
 from psyclone_test_utils import get_invoke
-from psyclone.transformations import TransformationError, ACCRoutineTrans
+from psyclone.transformations import TransformationError, ACCRoutineTrans, \
+    Dynamo0p3KernelConstTrans
```

Comment:
Please document the Dynamo0p3KernelConstTrans transformation in section 7.3.2 of the User Guide.

Reply:
Done (I think).